profiles/common/modules/custom/nexteuropa_lastupdate/theme/nexteuropa-last-update-block.tpl.php

```diff
@@ -5,8 +5,8 @@
  * Default implementation of the last update block.
  *
  * Available variables:
- * - $label: the language list.
- * - $date: optional language icon.
+ * - $label: the language label.
```

Comment:
:-1: It's not a language label. Last update label has nothing to do with language.
A good description of this variable may be "last updated label"